app/sections/videos/videos-list.component.ts

```diff
@@ -0,0 +1,51 @@
+import {Component} from 'angular2/core';
+import {RouteData} from 'angular2/router';
+import {Injector} from 'angular2/core';
+import {OnActivate, ComponentInstruction} from 'angular2/router';
+import {ContentfulService} from 'ng2-contentful/dist/src/services/contentful.service';
+import {ContentfulConfig} from '../../app.constans';
+import {ROUTER_DIRECTIVES} from 'angular2/router';
+import {transformResponse} from '../../shared/tools/response.tools';
+
+@Component({
+  template: <string> require('./videos-list.component.html'),
+  providers: [ContentfulService],
+  styles: [<string> require('./videos-list.component.scss')],
+  directives: [...ROUTER_DIRECTIVES],
+  host: {
+    class: 'content'
+  }
+})
+export class VideosList implements OnActivate {
+  private contentId: string;
+  private videos: any[];
+
+  constructor(private _routeData: RouteData,
+              private _contentful: ContentfulService,
+              _injector: Injector) {
+    let routeData = _injector.parent.parent.get(RouteData);
+    this.contentId = routeData.get('contentId');
+  }
+
+  routerOnActivate(to: ComponentInstruction, from: ComponentInstruction): any {
```

Comment:
As I already said - looks similar to about.component.ts. Encapsulation in separate service?